content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/accordion.html

```diff
@@ -0,0 +1,43 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe Systems Incorporated
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<div data-sly-use.accordion="com.adobe.cq.wcm.core.components.models.Accordion"
+     data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
+     class="cmp-accordion"
+     data-cmp-is="accordion">
+    <ol data-sly-test="${accordion.items && accordion.items.size > 0}"
+        class="cmp-accordion__tablist"
+        role="tablist"
+        aria-multiselectable="false">
+        <li data-sly-repeat.item="${accordion.items}"
+            class="cmp-accordion__item">
+            <div class="cmp-accordion__item--title${item.name == accordion.expandedItem ? ' initially-expanded' : ''}"
```

Comment:
According to accessibility practices the title of accordion should be wrapped with an element with role="button" (e.g `<button>`), which then should be wrapped in element that serves as a heading (e.g. `<h3>` element).

Also the heading in which the title is wrapped should probably be authorable to make sure, that heading structure throughout the page is correct.

sources:
https://davatron5000.github.io/a11y-nutrition-cards/components/accordion
https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html